### PR TITLE
fix(docs): syntax error in example code

### PR DESCRIPTION
**Pull request by yujunz**
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (please, look at the "Scope of the project" section in the README.md file)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

N/A

### CODE REVIEW

**Comment by yujunz** [comment]
Flake?

```
2020/03/25 06:44:05 could not resolve the version 'latest' to an actual version number: could not get releases from github.com/bazelbuild/bazel: could not download list of Bazel releases from github.com/bazelbuild: unexpected status code while reading https://api.github.com/repos/bazelbuild/bazel/releases: 403

Exited with code exit status 1
```

File: docs/install.md [hunk 1]
```diff
@@ -261,9 +261,11 @@ each.
 
 ```python
 workspace(
-    name = "my_wksp",
-    managed_directories = {"@app1_npm": ["app1/node_modules"],
-                           "@app2_npm": ["app2/node_modules"]}},
+    name = "maglev_ui",
```
Comment (alexeagle) [inline docs/install.md]: you did more than just fix the syntax error - you also changed the names of the repositories, now they don't match the *_install rules below

Comment (yujunz) [reply docs/install.md]: Fixed

Comment (alexeagle) [reply docs/install.md]: sorry, it isn't fixed - the names of the repos in managed_directories needs to match the names passed to the yarn_install and npm_install rules below.

just remove the spurious extra `}` in your PR, and optionally format it the way buildifier does

Comment (alexeagle) [reply docs/install.md]: I force-pushed a fix to your branch, will merge now. Thanks for pointing out the problem!

**Pull request merged**